design-editor/src/panel/property/attribute/attribute-element-image.js

```diff
@@ -169,31 +168,7 @@ class AttributeImage extends DressElement {
 	}
 
 	_onSrcImageChange(event) {
-		const self = this;
-		[].slice.call(event.target.files).forEach(file => {
-			const reader = new FileReader();
-			reader.addEventListener('loadend', event => {
-				if (event.target.readyState === FileReader.DONE) {
-					const
-						filePath = path.join('images', file.name),
-						writepath = pathUtils.createProjectPath(filePath),
-						readpath = pathUtils.createProjectPath(filePath, true);
-
-					utils.checkGlobalContext('writeFile') (
-						writepath,
-						event.target.result, {
-							encoding: 'binary'
-						},
-						() => {
-							AppManager.getActiveDesignEditor()
-								.getModel().updateAttribute(self._selectedElementId, 'src', readpath);
-							self._updateImageSourcePath(readpath);
-						}
-					);
-				}
-			});
-			reader.readAsBinaryString(file);
-		});
+		pathUtils.setImageSource(event, 'src', this._selectedElementId, this._updateImageSourcePath.bind(this));
```

Comment:
this line is quite long, maybe we should split it into separate lines like this:
```
pathUtils.setImageSource(
   event, 
   'src', 
   this._selectedElementId, 
   this._updateImageSourcePath.bind(this)
);
```